.ci/validation.sh

```diff
@@ -364,8 +365,8 @@ verify-regexp-id)
   fail=0
   for FILE in config/*_checks.xml
   do
-    a=$(grep -c "<module name=\"Regexp.*" $FILE) || a=0
-    b=$(grep "<module name=\"Regexp" -A 1 $FILE | grep -c "<property name=\"id\"") || b=0
+        a=$(grep -c "<module name=\"Regexp.*" "$FILE") || a=0
+        b=$(grep "<module name=\"Regexp" -A 1 "$FILE" | grep -c "<property name=\"id\"") || b=0
```

Comment:
Please restore indentation

Reply:
@romani Why restoration is needed and how we know that at which place, we need to restore the indentation?

Reply:
Changes Done